new/t4.php

```diff
@@ -0,0 +1,13 @@
+<?php 
+$x = 1; 
+while(TRUE) {
+    echo "The number is: $x <br>";
+    $x++;
+    break;
+} 
+
+WHILE($x <= 5) {
```

Comment:
PHP keywords must be lowercase; expected "while" but found "WHILE".

```suggestion
while($x <= 5) {
```